docs/database/server/server-v5.sql

```diff
@@ -54,7 +54,8 @@ CREATE TABLE [Channel] (
     [team_id] string  NOT NULL ,
     [type] string  NOT NULL ,
     [update_at] number  NOT NULL ,
-    [banner_info] string,
+    [banner_info] string  NOT NULL ,

```

Comment:
@harshilsharma63 I just noticed that quickDB added automatically the NOT NULL here, is it expected to be null?

Reply:
hey, yeah this should be nullable.

Reply:
ok, reverted the automatic change made by quickDB